packages/manager/modules/pci/src/components/project/images-list/images-list.controller.js

```diff
@@ -79,12 +79,16 @@ export default class ImagesListController {
     this.unavailableImages = {};
 
     forEach(this.os, (distributions, imageType) => {
-      this.availableImages[imageType] = {};
-      this.unavailableImages[imageType] = {};
+      // remove the prefix "baremetal-" if it exists to merge the 2 types
+      const formatedType = this.PciProjectImages.removeBaremetalPrefix(
+        imageType,
+      );
+      this.availableImages[formatedType] = {};
+      this.unavailableImages[formatedType] = {};
       forEach(distributions, (images, distribution) => {
```

Comment:
same as here:
https://github.com/ovh/manager/pull/7389/files#r900165944

Reply:
It doesn't work with the native forEach.

Reply:
I did it. I deleted lodash/forEach.